Sources/_StringProcessing/ByteCodeGen.swift

```diff
@@ -921,4 +880,51 @@ extension DSLTree.Node {
     default: return false
     }
   }
+
+  /// If the given node can be wrapped in a .quantify instruction
+  /// Currently this is conservative to reduce the coupling in ByteCodeGen between the normal case and
+  /// the quantified cases
+  ///
+  /// Essentially we trade off implementation complexity for runtime speed by adding more true cases to this
+  func shouldDoFastQuant(_ opts: MatchingOptions) -> Bool {
```

Comment:
// Future work: Should we allow ConsumeFunctions into .quantify?
      // This would open up non-ascii custom character classes as well as the
      // possibility of wrapping weirder cases into consume functions
      // (allowing us to .quantify anything we want, but increasing our reliance on ConsumerInterface)

Reply:
Ah right, in that case we'd need a separate `runQuantify` for consumers that would emit save points the normal way